### ISSUE TITLE
API access to weave

### ISSUE THREAD

**Issue by deadjoe** [weaveworks/weave#236]
are there any docs contains the way to access weave via REST API?

I have seen that there is a "HTTP_PORT" =6784 in the source code but can't see it from docker inspect after weave launched.

I am trying to write a testing code to operate weave via REST API. 

Thanks,

**Comment by avkonst** [comment]
Hi, is there rest API endpoint to get 'weave report' json data?

**Comment by bboreham** [comment]
@avkonst yes, although it's undocumented so it might change.  We [have discussed](https://github.com/weaveworks/weave/issues/2036) moving the endpoint from a TCP socket to a Unix socket, for instance.

Currently it's accessible via `curl 127.0.0.1:6784/report -H 'Accept: application/json'`

If you need it remotely accessible the same path will answer on the 'status' address, which you can set as described on the [metrics page](https://www.weave.works/docs/net/latest/metrics/)
